src/components/MapSubtitle.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <div id="subtitle">
+        <div id="subtitleText">
+          <p>An Indicator of Natural Water Storage</p>
+        </div>
+        <div id="subtitleInforButton">
+          <a href="#" class="icon">
+            <font-awesome-icon icon="info" />
+          </a>
+        </div>
+      </div>
+</template>
+
+<script>
+  export default {
+      name: 'MapSubtitle',
+      props: {
+          subTitle: {
+              type: String,
+              default: 'Add your title for the legend in MapBox.vue or make this blank'
```

Comment:
The way you have this setup I don't think you need the 'props' sections. If you want to include it so that the subtitle can be changed through a 'prop' (why don't they just say properties? I is really that much longer??), you can bind the 'prop' to 'MapSubtitle' component like it was done for the 'MapLegend' component:
 <MapLegend :legend-title="legendTitle" /> 
The ':' is Vue shorthand for 'v-bind:' which is the way you can tie the 'prop's to the component.

Reply:
Sorry, I pasted the code you would need in the comment above, but I was lost. So I guess you can just peek back at the code to see what I mean, argh.